src/dbt/kipptaf/models/performance_management/intermediate/int_performance_management__observation_details.sql

```diff
@@ -449,7 +449,7 @@ with
             1 as rn_submission,
         from historical_data as hd
         left join
-            {{ ref("base_people__staff_roster") }} as r
+            {{ ref("base_people__staff_roster_history") }} as r

```

Comment:
this need to be joined on dates as well or else you'll get dupes

Reply:
took it out completely